File: db/java/app/src/main/java/com/example/sales/domain/model/Warehouse.java
package com.example.sales.domain.model;

import java.time.LocalDateTime;

/**
 * 倉庫マスタのEntityクラス
 */
public class Warehouse {
    private String warehouseCode;
    private String warehouseName;
    private Integer warehouseType;
    private String address;
    private String phoneNumber;
    private String managerCode;
    private LocalDateTime createdAt;
    private String createdBy;
    private LocalDateTime updatedAt;
    private String updatedBy;

    // デフォルトコンストラクタ
    public Warehouse() {}

    // Getter/Setter（全フィールド）
    public String getWarehouseCode() { return warehouseCode; }
    public void setWarehouseCode(String warehouseCode) {
        this.warehouseCode = warehouseCode;
    }

    public String getWarehouseName() { return warehouseName; }
    public void setWarehouseName(String warehouseName) {
        this.warehouseName = warehouseName;
    }

    public Integer getWarehouseType() { return warehouseType; }
    public void setWarehouseType(Integer warehouseType) {
        this.warehouseType = warehouseType;
    }

    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }

    public String getPhoneNumber() { return phoneNumber; }
    public void setPhoneNumber(String phoneNumber) { this.phoneNumber = phoneNumber; }

    public String getManagerCode() { return managerCode; }
    public void setManagerCode(String managerCode) { this.managerCode = managerCode; }

    public LocalDateTime getCreatedAt() { return createdAt; }
    public void setCreatedAt(LocalDateTime createdAt) { this.createdAt = createdAt; }

    public String getCreatedBy() { return createdBy; }
    public void setCreatedBy(String createdBy) { this.createdBy = createdBy; }

    public LocalDateTime getUpdatedAt() { return updatedAt; }
    public void setUpdatedAt(LocalDateTime updatedAt) { this.updatedAt = updatedAt; }

    public String getUpdatedBy() { return updatedBy; }
    public void setUpdatedBy(String updatedBy) { this.updatedBy = updatedBy; }
}

File: db/go/sales_management/internal/model/warehouse.go
package model

import "time"

// Warehouse は倉庫マスタのドメインモデルです
type Warehouse struct {
	WarehouseCode string    `db:"倉庫コード"`
	WarehouseName string    `db:"倉庫名"`
	PostalCode    string    `db:"郵便番号"`
	Prefecture    string    `db:"都道府県"`
	Address1      string    `db:"住所１"`
	Address2      string    `db:"住所２"`
	PhoneNumber   string    `db:"電話番号"`
	FaxNumber     string    `db:"fax番号"`
	CreatedAt     time.Time `db:"作成日時"`
	CreatedBy     string    `db:"作成者名"`
	UpdatedAt     time.Time `db:"更新日時"`
	UpdatedBy     string    `db:"更新者名"`
}
